deploy-manage/autoscaling/trained-model-autoscaling.md

```diff
@@ -2,28 +2,188 @@
+
+## Enabling autoscaling in {{kib}} - adaptive resources [enabling-autoscaling-in-kibana-adaptive-resources]
+$$$nlp-model-adaptive-resources$$$
+
+You can enable adaptive resources for your models when starting or updating the model deployment. Adaptive resources make it possible for {{es}} to scale up or down the available resources based on the load on the process. This can help you to manage performance and cost more easily. When adaptive resources are enabled, the number of vCPUs that the model deployment uses is set automatically based on the current load. When the load is high, the number of vCPUs that the process can use is automatically increased. When the load is low, the number of vCPUs that the process can use is automatically decreased.
+
+You can choose from three levels of resource usage for your trained model deployment; autoscaling will occur within the selected level’s range.
+
+Refer to the tables in the [Model deployment resource matrix](#model-deployment-resource-matrix) section to find out the setings for the level you selected.
+
+:::{image} ../../images/machine-learning-ml-nlp-deployment-id-elser-v2.png
+:alt: ELSER deployment with adaptive resources enabled.
+:class: screenshot
+:::
+
+Search projects are given access to more processing resources, while Security and Observability projects have lower limits. This difference is reflected in the UI configuration: Search projects have higher resource limits compared to Security and Observability projects to accommodate their more complex operations.
+
+On Serverless, adaptive allocations are automatically enabled for all project types. However, the "Adaptive resources" control is not displayed in Kibana for Observability and Security projects.
+
+## Model deployment resource matrix [model-deployment-resource-matrix]
+
+The used resources for trained model deployments depend on three factors:
+
+* your cluster environment (Serverless, Cloud, or on-premises)
+* the use case you optimize the model deployment for (ingest or search)
+* whether model autoscaling is enabled with adaptive allocations/resources to have dynamic resources, or disabled for static resources
+
+If you use {{es}} on-premises, vCPUs level ranges are derived from the `total_ml_processors` and `max_single_ml_node_processors` values. Use the [get {{ml}} info API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ml-info) to check these values. The following tables show you the number of allocations, threads, and vCPUs available in Cloud when adaptive resources are enabled or disabled.
+
+::::{note}
+On Serverless, adaptive allocations are automatically enabled for all project types. However, the "Adaptive resources" control is not displayed in {{kib}} for Observability and Security projects.
+::::
+
+### Deployments in Cloud optimized for ingest [_deployments_in_cloud_optimized_for_ingest]
```

Comment:
I suggest merging the cloud and serverless sections and using tabs (e.g. a tab set under each H4, in two H3 sections - one of ingest and one for search). it will be easier to find what people are looking for. 

if these apply to elastic cloud hosted, and not some other meaning of cloud,  then you should adjust these titles. you should adjust them either way to make the scope clear.